tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_backward_ops.cpp

```diff
@@ -1666,7 +1687,7 @@ namespace tt::tt_metal::detail{
         )doc");
 
     m_tensor.def("binary_eq_bw", py::overload_cast<uint8_t, const Tensor&, const Tensor&, const Tensor&, const MemoryConfig&, const std::vector<bool>&, std::optional<Tensor>, std::optional<Tensor> >(&binary_eq_bw),
-            py::arg("queue_id").noconvert() = 0, py::arg("grad").noconvert(), py::arg("input").noconvert(), py::arg("other").noconvert(), py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, py::arg("are_required_outputs").noconvert() = std::vector<bool>{true, true}, py::arg("input_grad").noconvert() = std::nullopt,py::arg("other_grad").noconvert() = std::nullopt, R"doc(
+            py::arg("cq_id").noconvert() = 0, py::arg("grad").noconvert(), py::arg("input").noconvert(), py::arg("other").noconvert(), py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, py::arg("are_required_outputs").noconvert() = std::vector<bool>{true, true}, py::arg("input_grad").noconvert() = std::nullopt,py::arg("other_grad").noconvert() = std::nullopt, R"doc(
```

Comment:
Please change `cq_id` from a positional to a kw_only argument called `queue_id`.

CC @eyonland